k2/csrc/pytorch_context.cu

```diff
@@ -14,6 +14,43 @@
 #include "k2/csrc/pytorch_context.h"
 
 namespace k2 {
+// CAUTION: This is a workaround to free the CUDA memory
```

Comment:
It is fixed in
https://github.com/pytorch/pytorch/commit/394b720e38455c2c07e6a31ef816edd0c5afbeb4

But before k2 uses a PyTorch release that contains that fix, we will keep this workaround.